### PR TITLE
 Account may change when navigating between UIs#6151

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6151

*Description of changes:*
* append URL accountid in external URLs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by shabeeb-aot** [comment]
I added in some place, Let me know if i miss any links

File: auth-web/src/components/auth/home/NameRequestButton.vue [hunk 1]
```diff
@@ -23,9 +24,9 @@ export default class NameRequestButton extends Vue {
   // open Name Request in current tab to retain current account and user
   goToNameRequest (): void {
     if (LaunchDarklyService.getFlag(LDFlags.LinkToNewNameRequestApp)) {
-      window.location.href = ConfigHelper.getNameRequestUrl()
+      window.location.href = appendAccountId(ConfigHelper.getNameRequestUrl())
     } else {
-      window.location.href = ConfigHelper.getNroUrl()
+      window.location.href = appendAccountId(ConfigHelper.getNroUrl())
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/home/NameRequestButton.vue:29]: this can be removed.

File: auth-web/src/views/auth/home/RequestNameView.vue [hunk 2]
```diff
@@ -75,18 +76,18 @@ export default class RequestNameView extends Vue {
   // open Name Request in current tab to retain current account and user
   goToNameRequestExisting (): void {
     if (LaunchDarklyService.getFlag(LDFlags.LinkToNewNameRequestApp)) {
-      window.location.href = `${ConfigHelper.getNameRequestUrl()}existing`
+      window.location.href = appendAccountId(`${ConfigHelper.getNameRequestUrl()}existing`)
     } else {
-      window.location.href = `${ConfigHelper.getNroUrl()}nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1`
+      window.location.href = appendAccountId(`${ConfigHelper.getNroUrl()}nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1`)
```
Comment (sumesh-aot) [inline auth-web/src/views/auth/home/RequestNameView.vue:81]: this also can be removed

File: auth-web/src/views/auth/home/RequestNameView.vue [hunk 3]
```diff
@@ -75,18 +76,18 @@ export default class RequestNameView extends Vue {
...
   }
 
   // open Name Request in current tab to retain current account and user
   goToNameRequest (): void {
     if (LaunchDarklyService.getFlag(LDFlags.LinkToNewNameRequestApp)) {
-      window.location.href = ConfigHelper.getNameRequestUrl()
+      window.location.href = appendAccountId(ConfigHelper.getNameRequestUrl())
     } else {
-      window.location.href = ConfigHelper.getNroUrl()
+      window.location.href = appendAccountId(ConfigHelper.getNroUrl())
```
Comment (sumesh-aot) [inline auth-web/src/views/auth/home/RequestNameView.vue:90]: this too

**Review by sumesh-aot — APPROVED** [review]

File: auth-web/src/components/auth/home/NameRequestButton.vue (hunk 1 above)
Comment (shabeeb-aot) [reply auth-web/src/components/auth/home/NameRequestButton.vue:29]: can we remove LD flag check ?

**Pull request merged**